test/mpi/include/mpitestcxx.h

```diff
@@ -8,28 +8,27 @@
 #ifndef MPITESTCXX_H_INCLUDED
 #define MPITESTCXX_H_INCLUDED
 
-#ifndef MPITESTCONF_H_INCLUDED
-#error Required mpitestconf.h file not included first!
+#include "mpi.h"
+#include "mpitestconf.h"
+
+extern "C" {
+#include "mtest_common.h"
+}
+#ifdef HAVE_IOSTREAM
+#include <iostream>
+#ifdef HAVE_NAMESPACE_STD
+using namespace std;
+#endif
+#else
+#include <iostream.h>
 #endif
-
-#include <string.h>
-/*
- * Init and finalize test
- */
 void MTest_Init(void);
```

Comment:
`gnuindent` is having trouble dealing here, keep concatenating `void MTest_Init(void);` to the previous comment line.